package.json

```diff
@@ -20,6 +20,7 @@
     "heroku-postbuild": "gulp clean && gulp build --fortesting && gulp dist --fortesting"
   },
   "dependencies": {
+    "extend": "^3.0.0",
```

Comment:
yeah, (as visible here), we are reluctant about runtime deps.

Does `Object.extend` work for you?